src/views/Product/ShowDetails.vue

```diff
@@ -45,14 +54,15 @@ export default {
       id : null,
       token: null,
       isAddedToWishlist: false,
-      wishlistString:"Add to wishlist"
+      wishlistString:"Add to wishlist",
+      quantity: 1,
     }
   },
-  props : ["products", "categories"],
+  props : ["baseURL","products", "categories"],
   methods:{
     addToWishList(productId){
             console.log(this.isAddedToWishlist);
-            axios.post("http://remotedevs.org:8080/api/wishlist/add?token="+this.token, {
+            axios.post(this.baseURL+"wishlist/add?token="+this.token, {
```

Comment:
make it `${this.baseURL}wishlist/add?token=${this.token}` for better readability

Reply:
use '`'